test/functional/api/cas/cli_messages.py

```diff
@@ -23,6 +23,19 @@
     r"Error occurred, please see syslog \(/var/log/messages\) for details\."
 ]
 
+cache_start_failed = [
+    r"Error inserting cache \d+",
+    r"Old metadata found on device\.",
+    r"Please load cache metadata using --load option or use --force to",
+    r" discard on-disk metadata and start fresh cache instance\."
+]
+
+reinitialize_with_force_or_recovery = [
+    r"Old metadata found on device\.",
+    r"Please load cache metadata using --load option or use --force to",
+    r" discard on-disk metadata and start fresh cache instance\."
+]
```

Comment:
Should it be repeated? Isn't it better to split message and check for two separate? I think above message could be divided to even three messages and I bet one of them is passed to stdout, not stderr.

Reply:
I split messages as you suggested, good catch, thanks! I've double checked, they are both passed to stderr, test is passing;)